build_test.go

```diff
@@ -1345,18 +1345,6 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 				})
 
 				when("builder is untrusted", func() {
-					when("building Windows containers", func() {
-						it("errors and mentions that builder must be trusted", func() {
```

Comment:
is there another test in the section that tries to build Windows containers? If not, can we add one?

Reply:
These tests wouldn't exercise any new code paths due to mocking. Now that we've removed guard that returned an error for windows containers, there are no more windows-specific code paths.